File: AndroidStudioProjects/MyShoppingApp/MyShoppingApp/app/src/main/java/com/example/myshoppingapp/MainActivity.java
package com.example.myshoppingapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.SharedPreferences;
import android.os.Bundle;
import androidx.preference.PreferenceManager;
import android.util.Log;

public class MainActivity extends AppCompatActivity {

    public static ConfirmPurchaseDialog confirmPurchaseDialog = null;
    public static SharedPreferences preferences = null;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        preferences = PreferenceManager.getDefaultSharedPreferences(this);
        Cart.InitCart(preferences);

        confirmPurchaseDialog = new ConfirmPurchaseDialog(this);

        ShopDataManager.InitShopItems(findViewById(R.id.mainLayout), this);
    }
}

File: StudioProjects/Appointify/app/src/main/java/com/devsimone/appointify/MainActivity.kt
package com.devsimone.appointify

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.os.Looper
import androidx.appcompat.app.AppCompatActivity
import com.devsimone.appointify.databinding.ActivityMainBinding
import io.paperdb.Paper

class MainActivity : AppCompatActivity() {

    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        val handler = Handler(Looper.getMainLooper())
        handler.postDelayed({

            Paper.init(this@MainActivity)
                startActivity(Intent(this@MainActivity, LoginActivity::class.java))
                finish()

        }, 2000)
    }
}